proxy/src/main/java/org/apache/rocketmq/proxy/grpc/v2/channel/GrpcClientChannel.java

```diff
@@ -210,7 +211,7 @@ protected CompletableFuture<Void> processCheckTransaction(CheckTransactionStateR
     protected CompletableFuture<Void> processGetConsumerRunningInfo(RemotingCommand command,
         GetConsumerRunningInfoRequestHeader header,
         CompletableFuture<ProxyRelayResult<ConsumerRunningInfo>> responseFuture) {
-        if (!header.isJstackEnable()) {
+        if (Objects.isNull(header) || !header.isJstackEnable()) {
```

Comment:
can you explain why should this func just return when `header` is null?

Reply:
Failure to return will result in a null pointer error causing a large error log to be printed.